### PR TITLE
fix test issues

### PR DESCRIPTION
**Pull request by huminzhi**
@kchitalia @bhariharan @wmathurin @helenren   FYI, fixing test issues

### CODE REVIEW

File: SalesforceMobileSDK.xcworkspace/xcshareddata/xcschemes/Everything.xcscheme [hunk 1]
```diff
@@ -512,16 +512,6 @@
...
-               BlueprintIdentifier = "4F7EB4051BFFC88200768720"
-               BuildableName = "SalesforceSDKCoreTests.xctest"
-               BlueprintName = "SalesforceSDKCoreTests"
-               ReferencedContainer = "container:libs/SalesforceSDKCore/SalesforceSDKCore.xcodeproj">
-            </BuildableReference>
-         </TestableReference>
-         <TestableReference
-            skipped = "NO">
-            <BuildableReference
-               BuildableIdentifier = "primary"
```
Comment (huminzhi) [inline SalesforceMobileSDK.xcworkspace/xcshareddata/xcschemes/Everything.xcscheme]: This is a duplication, we have this SalesforceSDKCoreTests already..

File: libs/SmartSync/SmartSyncTestApp/Info.plist [hunk 2]
```diff
@@ -43,5 +43,25 @@
...
+			<key>salesforce.com</key>
+			<dict>
+				<key>NSExceptionRequiresForwardSecrecy</key>
+				<false/>
+				<key>NSIncludesSubdomains</key>
+				<true/>
+			</dict>
+		</dict>
+	</dict>
 </dict>
```
Comment (huminzhi) [inline libs/SmartSync/SmartSyncTestApp/Info.plist]: This is needed to fix SSL issue on ios9. (somehow the previous fix in SmartSyncTests-Info.plist doesn't work)

File: shared/test/test/SalesforceTestCase.swift [hunk 3]
```diff
@@ -46,7 +46,7 @@ class SalesforceTestCase: XCTestCase {
     
     func loginThroughUI() {
         // TODO move credentials to external file similar to test_credentials.json
-        loginDelegate.loginToSalesforce("sdktest@cs1.com", password:"test123456", host:Host.sandbox)
+        loginDelegate.loginToSalesforce("sdktest@cs1.com", password:"test1234", host:Host.sandbox)
     }
```
Comment (huminzhi) [inline shared/test/test/SalesforceTestCase.swift]: The pass was not right..

**Comment by bhariharan** [comment]
👍

**Comment by kchitalia** [comment]
👍

**Pull request merged**